py4cast/lightning.py

```diff
@@ -810,14 +863,34 @@ def validation_step(self, batch: ItemBatch, batch_idx):
 
         self.val_mean_loss = mean_loss
 
+        self.validation_step_logging(batch, prediction, target, mask)
+
+    def validation_step_logging(
+        self,
+        batch: ItemBatch,
+        prediction: NamedTensor,
+        target: NamedTensor,
+        mask: torch.Tensor,
+    ):
+        """
+        Save metrics and plots of validation to the tensorboard
+        """
```

Comment:
Docstring formating.
```suggestion
        """Saves metrics and plots of validation to the tensorboard
        """
```